extensions/interactions/TextInput/directives/text-input-rules.service.ts

```diff
@@ -29,51 +29,56 @@ import { TextInputRuleInputs } from 'interactions/rule-input-defs';
 export class TextInputRulesService {
   constructor(private nws: NormalizeWhitespacePipe) {}
   Equals(answer: TextInputAnswer, inputs: TextInputRuleInputs): boolean {
```

Comment:
Can you add new tests to explicitly cover the new boundary condition introduced here (no inputs), and the plural case (2 inputs) for each of the rules (8 tests in total)?

Reply:
Done